src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -259,6 +259,27 @@ CHIP_ERROR Instance::Read(const ConcreteReadAttributePath & aPath, AttributeValu
     case Attributes::FeatureMap::Id:
         return aEncoder.Encode(mFeatureFlags);
 
+    case Attributes::SupportedWiFiBands::Id:
+        if (mpDriver.Valid())
+        {
+            return aEncoder.Encode(mpDriver.Get<WiFiDriver *>()->GetSupportedWiFiBands());
```

Comment:
I suspect this will break in interesting ways in all-clusters-app, where the endpoint config doesn't know what sort of driver is being used, so we can end up in here with mpDriver being a thread driver even though SupportedWiFiBands is enabled....

We should fix that config issue, but it might be good to check instead of blindly dereferencing the Get() results.

Reply:
In that situation, How do I `check`? Is Checking feature map `Feature::kWiFiNetworkInterface` vs `Feature::kThreadNetworkInterface` the correct check? I believe this insures that it is the right instance of the driver used for the call right?

Reply:
Checking feature map would be good, yes.  That's what other things (e.g. HandleScanNetworks) do.